package.json

```diff
@@ -15,15 +15,15 @@
     "bump": "yakumo version",
     "dep": "yakumo upgrade",
     "pub": "yakumo publish",
-    "lint": "eslint --ext .tsx,.ts,.js .",
-    "format": "prettier --write .",
+    "lint": "eslint --ext .tsx,.ts,.js . --fix",
+    "lint:ci": "eslint --ext .tsx,.ts,.js . --max-warnings=0",
```

Comment:
In my opinion, warnings should be noticeable in the CI as well.
If there are some specific reasons that you want to bypass it, just use `@eslint-ignore-next-line` comments

Reply:
`--max-warnings 0` means that eslint will return 1 if there are more than 0 errors.

Reply:
Oh, that makes sense.